test/test_appscale_add_instances.py

```diff
@@ -133,11 +128,12 @@ def test_add_nodes_in_virt_deployment(self):
       LocalState.get_locations_json_location(self.keyname)).and_return(True)
 
     fake_nodes_json = flexmock(name="fake_nodes_json")
-    fake_nodes_json.should_receive('read').and_return(json.dumps([{
-      "public_ip" : "public1",
-      "private_ip" : "private1",
-      "jobs" : ["shadow", "login"]
-    }]))
+    fake_nodes_json.should_receive('read').and_return(json.dumps(
```

Comment:
fake_nodes_json.should_receive('read').and_return(json.dumps({
      "node_info":
        [
          {
            "public_ip": "public1",
            "private_ip": "private1",
            "jobs": ["shadow", "login"]
          }
        ]
    }))

@menivaitsi  like that?